packages/web-tests/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "web-tests",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test:dev": "playwright test",
+    "test:e2e": "start-server-and-test \"(cd ../../apps/web && npm run start)\" http://localhost:3000 \"playwright test\""
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
Not sure how licenses works, but is this the correct license?

Our packages seem to use AGPLv3/AGPL-3.0/MIT